src/NHSD.BuyingCatalogue.Identity.UserDatabase/Tables/Organisations.sql

```diff
@@ -1,8 +1,10 @@
 ﻿CREATE TABLE [dbo].[Organisations]
 (
 	Id UNIQUEIDENTIFIER NOT NULL,
-    [Name] NVARCHAR(255) NOT NULL,
+    [Name] NVARCHAR(255) NOT NULL,    
+    [Address] NVARCHAR(255) NULL,
```

Comment:
The address should probably be max. There are questions of whether or not this should be in its own table though. One for @michaelpace-bjss I think?